utils/check_copyright.py

```diff
@@ -41,11 +42,27 @@
            'Mostafa Ashraf',
            'Shiyu Liu',
            'ZHOU He']
-CURRENT_YEAR='2021'
+FIRST_YEAR = 2014
+CURRENT_YEAR = datetime.date.today().year
```

Comment:
This makes the build non-repeatable in a fundamental way.
I really really don't like doing that.  Like, really very much.

It's much more preferable to hardcode a date, or a hundred-date range, than to rely on a changeable condition like the current time.

Reply:
okay